src/base/bittorrent/session.cpp

```diff
@@ -1501,10 +1501,12 @@ void Session::enableBandwidthScheduler()
 void Session::populateAdditionalTrackers()
 {
     m_additionalTrackerList.clear();
-    for (QString tracker : asConst(additionalTrackers().split('\n'))) {
+
+    const QString trackers = additionalTrackers();
+    for (QStringRef tracker : asConst(trackers.splitRef('\n'))) {
```

Comment:
Are you sure it will be generally more effective if you still create new strings in the end?
IMO, splitting into QStringRef array really helps when you need to analyse it without creating strings from (almost) all string refs.

Reply:
In this case probably not much, but still a small improvement.
At least the QVector is denser and QStringRef itself and its operations is much lightweight than QString.

Reply:
Ok.